deployment/base/pvc.yaml

```diff
@@ -33,3 +33,21 @@ spec:
     requests:
       storage: 10Gi
   storageClassName: weekly-snapshots-retain-4 # provided by LTOps
+
+---
+
+# https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: fw-headless
+  namespace: languagedepot
+  labels:
+    app.kubernetes.io/part-of: languagedepot
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
```

Comment:
This seems really low to me. Aren't we putting tons of fwdata projects in here?

Reply:
Yeah, I'll bump it up. The /var/lib/languageforge folder on LF production is currently using 56G, and we expect to have more .fwdata files here. I'll make it 20Gi in base (and local-dev and develop), 40Gi in staging, and 80Gi in production.

Reply:
Done in commit acf0a274.